File: db4oj/core/src/com/db4o/internal/marshall/ObjectHeaderAttributes.java
/* Copyright (C) 2004 - 2006  db4objects Inc.  http://www.db4o.com */

package com.db4o.internal.marshall;

import com.db4o.*;
import com.db4o.internal.*;


/**
 * @exclude
 */
public abstract class ObjectHeaderAttributes {
    
    public abstract void addBaseLength(int length);
    
    public abstract void addPayLoadLength(int length);
    
    public abstract void prepareIndexedPayLoadEntry(Transaction trans);
    

}

File: db4o.net/Db4objects.Db4o/Db4objects.Db4o/Internal/Marshall/ObjectHeaderAttributes.cs
/* Copyright (C) 2004 - 2007  db4objects Inc.  http://www.db4o.com */

using Db4objects.Db4o.Internal;

namespace Db4objects.Db4o.Internal.Marshall
{
	/// <exclude></exclude>
	public abstract class ObjectHeaderAttributes
	{
		public abstract void AddBaseLength(int length);

		public abstract void AddPayLoadLength(int length);

		public abstract void PrepareIndexedPayLoadEntry(Transaction trans);
	}
}
